cmd/kops/BUILD.bazel

```diff
@@ -1,131 +1,11 @@
-load("@io_bazel_rules_go//go:def.bzl", "go_binary", "go_library", "go_test")
+load("@io_bazel_rules_go//go:def.bzl", "go_binary", "go_library")
```

Comment:
I'm not an expert in bazel. @justinsb is this good to remove?

Reply:
i think this is generated. i didn't explicitly removed it.